selfservice/flow/login/hook.go

```diff
@@ -125,6 +126,7 @@ func (e *HookExecutor) PostLoginHook(
 	f *Flow,
 	i *identity.Identity,
 	s *session.Session,
+	c *claims.Claims,
```

Comment:
It looks like you're passing nil here a lot. I'd prefer to see the functional option pattern ([Option 3: Functional Options Pattern](https://golang.cafe/blog/golang-functional-options-pattern.html)) in this case.

Just for the claims should be fine for now, as the other parameters are required and passed in, it seems.

Reply:
I pushed a change, does it look right to you?